Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Toy/speedloaders.yml

```diff
@@ -1,6 +1,6 @@
 - type: entity
   id: CapAmmoBase
-  name: "cap loader"
+  name: "cap gun loader"
```

Comment:
Probably doesn't need a name since its abstract and the inheritors should pick their own name.

Reply:
That's true since it's a base but was just keeping the name consistent with the new name for the item since it was named already.